ccd-definition/AuthorisationCaseField/CareSupervision/superuser.json

```diff
@@ -1153,5 +1153,12 @@
     "CaseFieldID": "draftSDOLinkDraftOrdersTab",
     "UserRole": "caseworker-publiclaw-superuser",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "warning_label",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "R"
```

Comment:
please update the userRole to `"caseworker-publiclaw-systemupdate"`. this entry should be in `system-update.json` file

Reply:
updated to "caseworker-publiclaw-systemupdate"